lib/views/items_details.erb

```diff
@@ -68,7 +68,7 @@
                 <input type="text" id="id" name="id" class="form-control" value="<%= @item_row[:id]%>"/>
                 <% end %>
               </div>
-              <div style="width:45%;" class="p-1">  
+              <div style="width:45%;" class="p-1">
```

Comment:
We should avoid inline css

Reply:
So add in an additional class or ID for these divs and then create a stylesheet to handle the widths for them outside of the erb file?